kube/pod.go

```diff
@@ -26,7 +26,7 @@ func NewPodTemplate(role *model.Role, settings ExportSettings, grapher util.Mode
 		return nil, fmt.Errorf("Role %s has no run information", role.Name)
 	}
 
-	vars, err := getEnvVars(role, settings.Defaults, settings.Secrets, settings)
+	vars, err := getEnvVars(role, settings.Defaults, settings)
```

Comment:
It seems weird that we are passing in `settings.Defaults` when we are also passing in `settings`. Given that this is the *only* call to this function (outside of tests), I think we should simplify to just pass in the `settings` object and let the function pick out the defaults itself.

Reply:
Will simplify.

Reply:
Addressed with commit 3fddb6f62a5f36b480ded5064693ef967ff36c86